translation: 0.3  # translational velocity
rotation: 0.3     # rotational velocity
trans_incr: 0.15   # Increase/Decrease translational velocity by this factor
rot_incr: 0.15     # Increase/Decrease rotational velocity by this factor